src/main/java/ca/carleton/controllers/AccountController.java

```diff
@@ -29,6 +30,9 @@
 
     private static final long MAX_API_REQUESTS = 1000;
 
+    @Autowired
```

Comment:
Instead of adding `@Autowired` here, you should add UserRepository to the AccountController constructor below.